### PR TITLE
Made some changes to the application.scss file

### PR DESCRIPTION
**Pull request by Princu7**
Solves the issue #815 .
@mariobehling  @aayusharora 
Here is the link to the gh-pages
https://princu7.github.io/websiteTest/speakers.html

### CODE REVIEW

**Comment by mariobehling** [comment]
The result of this change is, that in normal view the width of the dropdown speaker info is reduced. There is no reason to reduce the width in larger resolutions.

screenshot from 2016-10-04 14 17 46

Larger width as right now is fine (see below).

screenshot from 2016-10-04 14 19 26

**Comment by mariobehling** [comment]
A problem I see in this dropdown box is also that the line around the box is missing at the corners.

**Comment by Princu7** [comment]
@mariobehling Thanks for pointing out the issues. I will correct them right away.

**Comment by Princu7** [comment]
I made some changes. Please have a look at my gh-pages and review it. Thanks.
https://princu7.github.io/websiteTest/speakers.html

**Comment by mariobehling** [comment]
Thank you. On the top right and bottom right, the corner lines are still not completely closed. Is is possible to fix this?

**Comment by Princu7** [comment]
@mariobehling I am sorry. I tried hard to look for the cause of it but couldn't find it.  I have updated my pull request to the previous point( having the changes you mentioned). Suggest further course of action.

File: src/backend/_scss/application.scss [hunk 1]
```diff
@@ -481,7 +481,7 @@ a {
 .arrow {
   height: 10px;
   left: 6px;
-  margin-top: -2.3%;
+  margin-top: -9px;
```
Comment (aayusharora) [inline src/backend/_scss/application.scss]: Why it is not relative? Why to change this to px. Any reason?

Comment (Princu7) [inline src/backend/_scss/application.scss]: @aayusharora I am sorry. While working with other issues, the changes made in the gh-pages due to this pull request got lost and what you saw was the previous version without any alteration. I have updated my gh-pages to contain the changes done in this pull request.
The X-axis scroll bar is almost gone. It's still there but it's range is almost negligible.  Meaning that you can easily view the whole content without having to scroll to the right. To completely remove it we have to add the following properties to the code
```
body {
 overflow: -moz-scrollbars-vertical;
 overflow-x: hidden;
 overflow-y: scroll;
} 
```
I wasn't too sure about adding it so left it up for your review. About the margin values for the arrow, I tried different values both relative in percentages and absolute in pixels and the -10px seemed to work well on all the screens. We can change it if there is some issue with it. Thanks.
Please review again.
Link to updated gh-pages
https://princu7.github.io/websiteTest/speakers.html

**Comment by mariobehling** [comment]
@aayusharora Can this PR be merged?

**Comment by mariobehling** [comment]
Merging. Let's follow up on open questions in future PRs.

**Pull request merged**